README.md

```diff
@@ -45,3 +45,29 @@
 17. Open `Belt` globally. It saves a lot of typing.
 18. Prefer `Belt.Result` over throwing exceptions. This would make the execution flow more homogeneous. Exceptions are generally considered to be avoided nowadays.
 
+
+## Avo Specific Context
+
+
+### Cloud Functions
+
+When creating cloud functions, only define a single function in a file, defined as `let handle = {...}`
```

Comment:
Why not using trigger / https / pubsub instead of handle here as well?

Reply:
Handle is just to identify the entry of the function execution. If its the same name everywhere, it will become muscle memory to search it, instead of having to read the filename to check which type of function it is